yarn-project/accounts/src/testing/configuration.ts

```diff
@@ -45,7 +45,9 @@ export async function getDeployedTestAccountsWallets(pxe: PXE): Promise<AccountW
     INITIAL_TEST_SECRET_KEYS.filter(initialSecretKey => {
       const initialEncryptionKey = sha512ToGrumpkinScalar([initialSecretKey, GeneratorIndex.IVSK_M]);
       const publicKey = generatePublicKey(initialEncryptionKey);
-      return registeredAccounts.find(registered => registered.publicKey.equals(publicKey)) != undefined;
+      return (
+        registeredAccounts.find(registered => registered.masterIncomingViewingPublicKey.equals(publicKey)) != undefined
```

Comment:
We currently use masterIncomingViewingPublicKey all around TS so if we decide to rename it I would do that in a separate PR. I originally didn't do it because TS lint complains about camel case. See [this discussion](https://aztecprotocol.slack.com/archives/C04PUD9AA4W/p1714758791915599).